dishes/dishes-model.js

```diff
@@ -0,0 +1,36 @@
+const db = require('../data/dbConfig.js');
+
+module.exports = {
+  getDishes,
+  addDish,
+  getDish,
+  getRecipes,
+  addRecipe,
+};
+
+function getDishes() {
+  return db('dishes');
+}
+
+function addDish(dish) {
+  return db('dishes').insert(dish);
```

Comment:
Your FE will probably want a response with the object you just added. That's between you and the FE, though. If they're just routing to a page that will fetch dish by id anyways, then this is fine